src/core_atmosphere/physics/mpas_atmphys_manager.F

```diff
@@ -547,11 +551,13 @@ subroutine physics_run_init(configs,mesh,state,clock,stream_manager)
  endif
 
 !set alarm for updating the background surface albedo and the greeness fraction:
- call mpas_set_timeInterval(alarmTimeStep,timeString=config_greeness_update,ierr=ierr)
- alarmStartTime = startTime
- call mpas_add_clock_alarm(clock,greenAlarmID,alarmStartTime,alarmTimeStep,ierr=ierr)
+ if (config_gvf_update) then
```

Comment:
@tanyasmirnova this if block seems unnecessary now and maybe even not working correctly if you want to update albedo and not vegetation, in that case greenAlarmID needs to be set and ringing

Reply:
@barlage To begin with, it is a very bad set-up to do albedo and greenness updating connected to each other. Our situation proves it not robust, because we treat greenness in a special way. I would rewrite surface updating but as you said there wil be more issues with merging in future.

Reply:
I chose the least invasive solution to get the real-time greenness vegetation fraction from the parent model to MPAS. I see some problems in MPAS with surface updating, but my understanding that we are not supposed to change any of their code, it is "untouchable" because of future commits. Therefore, I added a new config_gvf_update parameter to achieve our goal.

> @tanyasmirnova this if block seems unnecessary now and maybe even not working correctly if you want to update albedo and not vegetation, in that case greenAlarmID needs to be set and ringing

I have removed the if(config_gvf_update) from line 554. So - the "green" alarm will be still created, but I pass config_gvf_update into physics_update_surface, and when it is set to false, the GVF will not be overwritten with climatology, but background albedo will be created.  At initial forecast time surface_update repeats what was already done in mpas_atmphys_initialize_real.F at the INIT step. Therefore, background albedo was already computed. And we do not want update the background albedo in our MPAS runs. The actual albedo will be updated for grid points covered with snow. In RRFS we also update it for low solar angles, but so far not in MPAS.

Reply:
@barlage Mike, could you please revisit my PR. I think I took into account all your comments and the PR should be ready to merge. Thank you.